webui/src/Buttons/EditButton/FeedbackPanel.jsx

```diff
@@ -283,6 +316,7 @@ export function FeedbackEditor({ feedback, setValue, innerDelete, setSelectedSty
 								actionId={feedback.id}
 								value={(feedback.options || {})[opt.id]}
 								setValue={setValue}
+								visibility={optionVisibility[opt.id] === false ? 'none' : null}
```

Comment:
I think it would be better for the visibility prop to be a boolean rather than a css value.
If you want to leave it like this that is fine too, Im thinking for future proofing for when we want to change the rendering and how we make them hidden changes

Reply:
That makes sense.